app/gitcoinbot/gitcoinbotActions.py

```diff
@@ -0,0 +1,133 @@
+"""
```

Comment:
Would you mind renaming this file to `actions.py`, `helpers.py`, or `utils.py` for consistency?